examples_gbs/run_tutorial_sample.py

```diff
@@ -0,0 +1,140 @@
+# a look at this by using a small fixed graph as an example:
+
+from strawberryfields.gbs import plot
+import networkx as nx
+import plotly
+
+adj = np.array(
+    [
+        [0, 1, 0, 0, 1, 1],
+        [1, 0, 1, 0, 1, 1],
+        [0, 1, 0, 1, 1, 0],
+        [0, 0, 1, 0, 1, 0],
+        [1, 1, 1, 1, 0, 1],
+        [1, 1, 0, 0, 1, 0],
+    ]
+)
+
+graph = nx.Graph(adj)
+
+plotly.offline.plot(plot.plot_graph(graph), filename="random_graph.html")
+
+##############################################################################
+# .. raw:: html
+#     :file: ../../examples_gbs/random_graph.html
+#
+# We can sample this graph from GBS using:
+
+n_mean = 4
+samples = 20
+
+s = sample.sample(adj, n_mean, samples)
+
+print(s[:5])
+
+##############################################################################
+# Each sample in ``s`` is a list of modes with ``1``'s for nodes that have clicked and ``0``'s
+# for nodes that haven't. We want to convert a sample to another representation where the result
+# is a list of modes that have clicked. This list of modes can be used to select a subgraph of
+# the graph. For example, if ``[0, 1, 0, 1, 1, 0]`` is a sample from GBS then ``[1, 3, 4]`` are
+# the selected nodes of the corresponding subgraph.
+#
+# However, the number of clicks in GBS is a random variable and we are not always guaranteed to
+# have enough clicks in a sample for the resultant subgraph to be of interest. We can filter out
+# the uninteresting samples using:
+
+min_clicks = 3
+max_clicks = 4
+
+s = sample.postselect(s, min_clicks, max_clicks)
+
+print(len(s))
+s.append([0, 1, 0, 1, 1, 0])
+
+##############################################################################
+# As expected, we can see that we have lost some samples. The number of samples that survive this
+# postselection is determined by the mean photon number in GBS. We have also added in our example
+# sample ``[0, 1, 0, 1, 1, 0]`` to ensure that there is at least one for the following.
+#
+# Let's convert our postselected samples to subgraphs:
+
+subgraphs = sample.to_subgraphs(s, graph)
+
+print(subgraphs)
+
+##############################################################################
+# We can take a look at one of the sampled subgraphs:
+
+plotly.offline.plot(plot.plot_graph(graph, subgraphs[0]), filename="subgraph.html")
+
+##############################################################################
+# .. raw:: html
+#     :file: ../../examples_gbs/subgraph.html
+#
+# These sampled subgraphs act as the starting point for some of the applications made available
+# in Strawberry Fields, including the maximum clique and dense subgraph identification problems.
+# Go and check out the other tutorials in the applications layer to see what you can do with GBS!
+#
+# .. note::
+#       Simulating GBS can be computationally intensive when using both threshold and PNR
+#       detectors. After all, it is a quantum algorithm! To help users get to grips with the

```

Comment:
Maybe: "After all, we are using a classical algorithm to simulate a quantum process! ..."